mods/mc_worldmanager/init.lua

```diff
@@ -32,11 +32,9 @@ mc_worldManager.load_data()
 function mc_worldManager.GetSpawnRealm()
     local spawnRealm = Realm.realmDict[mc_worldManager.spawnRealmID]
     if (spawnRealm == nil) then
-        spawnRealm = Realm:New("Spawn Realm", 80, 80)
+        spawnRealm = Realm:NewFromSchematic("Spawn Realm", "vancouver_osm")
```

Comment:
TODO: Make spawn realm selection dynamic and create a prompt for the teacher to select a schematic from a dropdown menu or use a mapgen on initiation of a `Minetest_Classroom` world.